h/views/client.py

```diff
@@ -20,11 +20,21 @@
 DEFAULT_CLIENT_URL = "https://cdn.hypothes.is/hypothesis"
 
 
+def process_url_template(url_template, request):
+    """
+    Replace placeholders in a URL with elements of the current request's URL.
+    """
+    url = url_template.replace('{current_host}', request.domain)
+    url = url.replace('{current_scheme}', request.scheme)
+    return url
```

Comment:
This function should really live somewhere else as it is used by other modules. Suggestions welcome.

Reply:
It looks like it's only used by views, so following the new "local helpers" (rather than global util dir) patterns that's now used exclusively in lms and partially in h, the lazy place to put this would be `h/views/helpers.py`

Reply:
I think "render" is more appropriate than "process" for "rendering a template"

Reply:
I agree, I've renamed it.

Reply:
Regarding location, I realized that we already had an `h.util.view` module and so, for consistency within this project, I put the `render_url_template` function there. That entire module can then be moved to `h.views.helpers` if you wish in future.

Reply:
:+1: